src/core.py

```diff
@@ -114,7 +115,6 @@ def on_config(self, config):
         config["markdown_extensions"].append("pymdownx.mark")
         config["markdown_extensions"].append("pymdownx.smartsymbols")
         config["markdown_extensions"].append("pymdownx.snippets")
-        config["markdown_extensions"].append("pymdownx.superfences")
```

Comment:
Removing here since `pymdownx.superfences` is already supplied from `pymdownx.extra`